src/content/en/updates/2017/04/headless-chrome.md

```diff
@@ -335,13 +343,40 @@ Demos
 
 ## FAQ
```

Comment:
I'll approve anyway. You really need to start working your answers into the article.